test/check-review

```diff
@@ -32,7 +32,11 @@ class TestReview(anacondalib.VirtInstallMachineCase):
         m = self.machine
         i = Installer(b, m)
         s = Storage(b, m)
-        r = Review(b)
+        r = Review(b, m)
+
+        # Set default hostname for check
```

Comment:
What happens if you dont set that? Does the backend have some other value autoset?

Reply:
Backend default value is empty value. So hostname will be empty.